doc/production-hosting.md

```diff
@@ -247,12 +177,3 @@ Request performance is monitored in two places:
 [monitor]: https://shields.redsparr0w.com/1568/
 [notifications]: http://shields.redsparr0w.com/discord_notification
 [monitor discord]: https://discordapp.com/channels/308323056592486420/470700909182320646
-
-## Known limitations
-
-1.  The only way to inspect the commit on the server is with `git ls-remote`.
```

Comment:
This list item, about `git ls-remote` probably should be kept with the legacy servers until they are no longer part of production.